doc/man/8/rbdmap-generator.rst

```diff
@@ -0,0 +1,140 @@
+:orphan:
+
+============================================================================
+ rbdmap-generator -- Systemd unit generator for map RBD devices at boot time
+============================================================================
+
+.. program:: rbdmap-generator
+
+Synopsis
+========
+
+| **/lib/systemd/system-generators/rbdmap-generator**
+
+
+Description
+===========
+
+**rbdmap-generator** is a generator that translates /etc/ceph/rbdmap into native
+systemd units early at boot and when configuration of the system manager is
+reloaded. This will create rbdmap@.service units as necessary. When these units
+are activated or deactivated, ``rbd map`` or ``rbd unmap`` command is called as
+appropriate.
+
+**rbdmap-generator** implements **systemd.generator**\(7)
+
+The /etc/ceph/rbdmap file describes RBD devices that are set up during system
+boot.
+
+Empty lines and lines starting with the "#" character are ignored. Each of the
+remaining lines describes one encrypted block device. Fields are delimited by
+white space.
+
+Each line is in the form::
+
+    image-or-snap-spec options
+
+#. image-or-snap-spec is the image or snapshot specification, in the form of
+   [<pool-name>/[<namespace>/]]<image-name>[@<snap-name>]. If only <image-name>
+   is specified, <pool-name> defaults to "rbd".
+
+#. options is a comma-delimited list of options. This field is optional. The
+   supported options are listed below.
+
+If successful, the generated unit maps the image to a ``/dev/rbdX`` device, at
+which point a udev rule is triggered to create a friendly device name symlink,
+``/dev/rbd/<image-or-snap-spec>``, pointing to the real mapped device.
+
+When writing ``/etc/fstab`` entries for RBD images, The _netdev mount option
+should be used. Otherwise, a dependency loop might be created where the mount
+point will be pulled in by local-fs.target, while the service to configure the
+network is usually only started after the local file system has been mounted.
+
+Supported Options in Config File
+--------------------------------
+
+.. option:: conf=, cluster=, id=, name=, mon_host=, keyfile=, keyring=, device-type=, read-only, exclusive, quiesce=, quiesce-hook=
+
+   These options are passed to :doc:`rbd <rbd>`\(8) command directly as
```

Comment:
```suggestion
   These options are passed to :program:`rbd`\(8) command directly as
```

as `rbd` is also defined as under the "program" domain, and can be referenced by `:program:` directly.